Feature/migrate to vuepress

Dragonesis opened the pull request:
Finish

Dragonesis committed:
Delete file

Dragonesis committed:
Add project to vuepress

stcherenkov commented on .vuepress/src/components/ProjectLogo/ProjectLogo.vue:
```diff
@@ -0,0 +1,19 @@
+export default {
+  name: 'ProjectLogo',
+  props: {
+    color: 'color' | 'white'
```
Did you mean `'colored'`?

stcherenkov replied:
And also property name, smth like `theme` maybe?

stcherenkov commented on Dockerfile:
```diff
@@ -1,13 +0,0 @@
-FROM node:8-onbuild AS build
```
These removed files are removed temporary?

Dragonesis replied:
Yes

stcherenkov commented on news/news2.md:
```diff
@@ -0,0 +1,67 @@
+img2x: /assets/img/content/cover/zawya@2x.png
+---
+
+<NewsPage> 
```
Let's remove this wrapper. We already put this file under `/news` – so it is not something else, for sure.

stcherenkov commented:
Ok, looks nice, but for full assessment needs an example of i18n – both for news-like pages and for landing-like pages.

Dragonesis committed:
Add partners card

Dragonesis committed:
Change location

Dragonesis committed:
Moved sprite bundle in SpriteIcon, add check for page type

Dragonesis committed:
Update support files

Dragonesis committed:
Update news components

Dragonesis committed:
Moved cover in folder public

Dragonesis committed:
Four real news added

IViol commented on .vuepress/src/components/FooterMenu/FooterMenu.vue:
```diff
@@ -0,0 +1,66 @@
+    <div class='box' data-aos='fade-down' data-aos-duration='300' data-aos-delay="150">
+      <div class='title'>Company</div>
+      <div class='item '>
+        <a href='about.html' class='link ' >
```
I assume, that now we can remove `.html` from links

IViol commented on .vuepress/theme/Layout.vue:
```diff
@@ -0,0 +1,64 @@
+        return 'color';
+      }
+    },
+    getTypePgae() {
```
getTypePgae -> getTypePage

IViol replied:
is it ok to place this file within `.vuepress` dir?

Dragonesis replied:
This folder is assumed in VuePress itself

IViol reviewed (commented):
- please setup/update linters
- some of necessary for deployment files were removed, need to re-check them

Dragonesis committed:
Add localization json

Dragonesis committed:
Add сommon styles

Dragonesis committed:
Add CareersPage template

Dragonesis committed:
Update navigations component, modalToggle partition

Dragonesis committed:
Add list vacancies and JobInfo

Dragonesis committed:
Corrected name props for coloristic layout

Dragonesis committed:
Add modal component

Dragonesis committed:
Update news previews and news page

Dragonesis committed:
Add contects list

Dragonesis committed:
Add suport box

Dragonesis committed:
Add vacancy page

Dragonesis committed:
Add project form

Dragonesis committed:
Update project logo

Dragonesis committed:
Working language switch

Dragonesis committed:
Add not found page

Dragonesis committed:
Update Layout

Dragonesis committed:
Add content page

Dragonesis committed:
Main page

Dragonesis committed:
Update config and package.json

Dragonesis committed:
Remove HTML postfixes for links

Dragonesis committed:
Change path to public img

Dragonesis committed:
Transferring pages markup to files in the folder 'pages'

Dragonesis committed:
Repositioning components for better structure

Dragonesis committed:
All news added

Dragonesis committed:
Replacing the slider "swiper" on "Glade"

Dragonesis committed:
Temporary shutdown aos

Dragonesis committed:
Return server files

Dragonesis committed:
Add mobile fooert and global init vue-meta

Dragonesis committed:
Intagrated meta info

Dragonesis committed:
Small fix

Dragonesis committed:
Add mobile footer and change place CircleSocial

Dragonesis committed:
Add favicon set

Dragonesis committed:
Update config

Dragonesis committed:
Fix meta markup

Commit:
Update .gitignore

Commit:
Fix versions

Remove ^ from versions in package.json
Add .npmrc

Dragonesis committed:
Include gtm and ym

IViol commented on careers/ux-analyst-and-designer.md:
```diff
@@ -0,0 +1,51 @@
+---
+published: true
+title: UX Analyst & Designer
+description: Jibrel is looking for a Middle Python Developer to join its Saint-Petersburg office. Successful candidate will develop backend applications that interact with the different blockchains.
```
wrong description

Dragonesis committed:
Test icon

Dragonesis committed:
Fix

Commit:
Update package meta

Commit:
Update news

Simplify page meta
Fix twitterwidget issues
Introduce figure for block-like images
Rename news sources images

Commit:
Fix images alts

Dragonesis committed:
Base icon config and SpriteIcon component

Dragonesis committed:
Implement SpriteIcon componet

Dragonesis committed:
Corrected some icon

Dragonesis committed:
Fixed conflict

Dragonesis committed:
Add SpriteIcon for page

Dragonesis committed:
Small fix

Dragonesis committed:
Update news for korean language

Commit:
Fix production build

stcherenkov committed:
Merge branch 'develop' into feature/migrate-to-vuepress

Commit:
Fix icons for production

Commit:
Merge remote-tracking branch 'origin/feature/migrate-to-vuepress' into feature/migrate-to-vuepress

Commit:
Fix favicons

Commit:
Fix opengraph metadata

Commit:
Update version.txt
